content/workshops/example/workshop_content.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Workshop Content
+workshops: example
+workshop_weight: 30
+layout: lab
+---
+
+## SETTING EXPECTATIONS FOR YOUR AUDIENCE
+
+----
```

Comment:
Don't put horizontal lines as it doesn't match the style of the rest of the labs in the example workshop.